app/services/tims.rb

```diff
@@ -4,7 +4,7 @@
 class Tims
 
   def get_disruptions
-    download_and_filter_disruption.map{ |disruption| Disruption.new(*first_divided(disruption["CauseArea"]["DisplayPoint"]["Point"].values)) }
+    download_and_filter_disruption.map{ |disruption| Disruption.new(*disruption["CauseArea"]["DisplayPoint"]["Point"]["coordinatesLL"].split(",").map(&:to_f)) }
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.